src/main.ts

```
@@ -37,6 +38,8 @@ async function bootstrap() {
 		}),
 	)
 
+	app.use(new RequestPathLoggerMiddleware().use);
```

ESLint

[error] Delete `;`

(prettier/prettier)